### PR TITLE
Unit test phg clikt

### PR DESCRIPTION
**Pull request by tcasstevens**
## Description

Added unit test for Phg.kt to increase code coverage

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated relevant documentation

## Changelog entry

```release-note
Added unit test for Phg.kt to increase code coverage
```

### CODE REVIEW

**Review by lynnjo — APPROVED** [review]

File: src/main/kotlin/net/maizegenetics/phgv2/cli/Phg.kt [hunk 1]
```diff
@@ -19,7 +20,16 @@ class Phg : CliktCommand() {
         var minorVersion = 0
         var patchVersion = 0
         var buildNumber = 0
-        Phg::class.java.getResourceAsStream("/version.properties").bufferedReader().readLines().forEach {
+
+        val reader = try {
```
Comment (btmonier) [inline src/main/kotlin/net/maizegenetics/phgv2/cli/Phg.kt:24]: Could provide some comments for what this new code is doing?

**Review by btmonier — APPROVED** [review]

**Comment by btmonier** [comment]
Could you also add information to the PR template (e.g., add a changelog entry) for future reference?

**Comment by tcasstevens** [comment]
> Could you also add information to the PR template (e.g., add a changelog entry) for future reference?

Ok, I think I did what you wanted?

File: src/test/kotlin/net/maizegenetics/phgv2/cli/PhgTest.kt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    @Test
+    fun testPhgCommand() {
+        val result = Phg().test(
+            "--help"
+        )
+
+        myLogger.info("testPhgCommand: result output: ${result.output}")
+
+        assertEquals(result.statusCode, 0, "status code not 0: ${result.statusCode}")
```
Comment (zrm22) [inline src/test/kotlin/net/maizegenetics/phgv2/cli/PhgTest.kt:22]: You should probably check the output coming from the result and make sure it matches the expected.

**Review by zrm22 — APPROVED** [review]

**Pull request merged**